permissions/plugin-zerobug.yml

```
@@ -0,0 +1,7 @@
+--- 
+name: "zerobug"
+github: "jenkinsci/zerobug-plugin"
+paths:
+- "org/jenkins-ci/plugins/zerobug"
```

```suggestion
- "io/jenkins/plugins/zerobug"
```

https://github.com/engfelipeoliveira/zerobug-plugin/blob/master/pom.xml#L10